docs/en/observability/view-infrastructure-metrics.asciidoc

```diff
@@ -63,7 +63,7 @@ image::images/pod-metrics.png[Kubernetes pod metrics]
 [[apm-uptime-integration]]
 == Integrate with Logs, Uptime, and APM
 
-Depending on the features you have installed and configured, you can view logs, traces, or uptime information relating to a specific resource.
+Depending on the features you have installed and configured, you can view logs, traces, or Uptime information relating to a specific resource.
```

Comment:
Same comments are before, should this be capitalized?

Reply:
No need to. I would say it's safe to close this PR.